src/main/kotlin/me/partlysanestudios/partlysaneskies/features/chat/ChatTransformer.kt

```diff
@@ -0,0 +1,51 @@
+//
+// Written by DerGruenkohl
+// See LICENSE for copyright and license notices.
+//
+
+
+package me.partlysanestudios.partlysaneskies.features.chat
+
+import cc.polyfrost.oneconfig.events.event.ChatSendEvent
+import cc.polyfrost.oneconfig.libs.eventbus.Subscribe
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies.Companion.config
+
+
+//Currently only using to owoify send chat messages (meow)
+object ChatTransformer {
+    var lastmsg = ""
+
+    @Subscribe
+    fun onChat(event: ChatSendEvent) {
+        //dont mess with chats that dont want to
+        if (!doChatTransform()) return
+
+        event.isCancelled = true
+        val msg = event.message
+        //Do not go into recursion (bad)
+        if (lastmsg == msg) {
+            event.isCancelled = false
+            return
+        }
+        //dont break commands :)
+        if (msg.startsWith("/")) return
```

Comment:
More of a petty style thing for debugging, I'd rather not do this but it's fine

Reply:
I should be more specific. I mean like 

```kt
 if (msg.startsWith("/")) {
    return
}
 ```

Reply:
I see. Its better when the whole codebase uses the same format for these checks